RevenueCatUI/Data/PaywallViewConfiguration.swift

```diff
@@ -28,7 +29,8 @@ struct PaywallViewConfiguration {
         fonts: PaywallFontProvider = DefaultPaywallFontProvider(),
         displayCloseButton: Bool = false,
         introEligibility: TrialOrIntroEligibilityChecker? = nil,
-        purchaseHandler: PurchaseHandler? = nil
+        purchaseHandler: PurchaseHandler? = nil,
+        isDarkMode: Bool? = nil
```

Comment:
Let's inject a `UIUserInterfaceStyle?` and then we can convert that internally to `ColorScheme`.

Reply:
https://github.com/RevenueCat/purchases-ios/pull/3647/commits/112eec4d8863affb43e31487e0a3d59aaac6f99d

thanks for the feedback, @NachoSoto i've updated the PR here. Also removed the isDarkMode boolean to accept a style from the viewcontroller